shopinvader/models/shopinvader_variant.py

```diff
@@ -242,11 +243,34 @@ def _get_price_per_qty(self, qty, pricelist, fposition, company=None):
         return res
 
     def _compute_main_product(self):
+        # Respect same order.
+        order_by = self.env["product.product"]._order.split(",")
+        fields_to_read = ["tmpl_record_id"] + order_by
+        tmpl_ids = self.mapped("tmpl_record_id").ids
+        # Use sudo to bypass permissions (we don't care)
+        _variants = self.sudo().search(
+            [("tmpl_record_id", "in", tmpl_ids)], order="tmpl_record_id"
+        )
+        # Use `load=False` to not load template name
+        variants = _variants.read(fields_to_read, load=False)
```

Comment:
is read faster? it will not use the cache

Reply:
it's quite fast as I read only 4 fields and load=false -> no join on the template